Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: doc
+doc:
+	docker run -t --rm -e "DOCS_USER=`id -u`:`id -g`" -v `pwd`/Makefile:/Makefile:ro -v `pwd`/docs:/docs:ro -v `pwd`/docs/_build:/docs/_build python:3.6 make build
+
+.PHONY: build
+build:
+	pip install docutils sphinx
```

Comment:
Installing the requirements file should be enough.

Reply:
Note: I tried that and got the error: ModuleNotFoundError: No module named 'docutils'
This is strange because the requirment.txt is enough on Travis and in my local python virtualenv.

Reply:
Can we move the `build` to `Dockerfile` and `doc` to `docker-compose.yml`? They are natural places for Docker workflow. I would be confused seeing Makefile in docs which uses docker for builds.

PS: @erbou could we also integrate your script which uses the `virtualenv`?

Reply:
Don't you still need some external script (like Makefile) to extract the docs after they are built?

I'm just posting this 'as-is' is; it worked on my laptop to check if my changes to the docs were correct.